### PR TITLE
feat(unlock-app) - add explicit button to scan next ticket

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
Add ability to scan next ticket from the UI when ticket is already marked as check-in 

# Issues

Fixes #
Refs #
Screenshot 2022-07-13 at 19 14 03

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/interface/VerificationStatus.tsx [hunk 1]
```diff
@@ -18,13 +18,14 @@ import { isSignatureValidForAddress } from '~/utils/signatures'
 
 interface Props {
   config: MembershipVerificationConfig
+  setConfig: (config: MembershipVerificationConfig | null) => void
```
Comment (searchableguy) [inline unlock-app/src/components/interface/VerificationStatus.tsx:21]: Rename this to generic onScanNext function. We should expose assume anything about how the parent deals with state above. Let them pass a generic function which is called when x happens.

File: unlock-app/src/components/interface/verification/Scanner.tsx [hunk 2]
```diff
@@ -96,7 +96,10 @@ export function Scanner() {
               <div className="fixed p-6 inset-0 overflow-y-auto">
                 <div className="flex min-h-full items-center justify-center text-center">
                   <Dialog.Panel className="max-w-sm w-full">
-                    <VerificationStatus config={membershipVerificationConfig} />
+                    <VerificationStatus
+                      setConfig={setMembershipVerificationConfig}
```
Comment (searchableguy) [inline unlock-app/src/components/interface/verification/Scanner.tsx:100]: Should be `onVerified={() => { setMembershipVerificationConfig(null) }}`

**Review by searchableguy — CHANGES_REQUESTED** [review]
You need to deal with how redirect on the verification page as well here.

**Comment by julien51** [comment]
> You need to deal with how to redirect on the verification page as well here.

Can you clarify please?

**Comment by julien51** [comment]
> Outside of scanner, the verification page which shows the status when you use the full URL. We should have a scan button there well, right?

Makes sense and yes! Thank you!

**Review by julien51 — APPROVED** [review]

File: unlock-app/src/components/interface/VerificationStatus.tsx [hunk 3]
```diff
@@ -154,24 +155,39 @@ export const VerificationStatus = ({ config }: Props) => {
 
   const checkedInAt = membershipData?.metadata?.checkedInAt
 
+  const disableActions =
+    !isVerifier || isCheckingIn || !!invalid || !!checkedInAt
+
+  const onVerifiedCb = () => {
+    console.log('onVerifiedCb', onVerified)
```
Comment (searchableguy) [inline unlock-app/src/components/interface/VerificationStatus.tsx:162]: Remove

**Pull request merged**